src/helper_functions.py

```diff
@@ -31,45 +39,41 @@ def load_data_msci(path: str = None, n: int = 24) -> Dict[str, pd.DataFrame]:
     df.index = pd.to_datetime(df.index, format='%d/%m/%Y')
     series_id = df.columns[0:n]
     X = df[series_id]
-    
+
     # Load msci world index return series
     y = pd.read_csv(f'{path}NDDLWI.csv',
                          sep=';',
                          index_col=0,
                          header=0,
                          parse_dates=True)
```

Comment:
Plz fix indentation

Reply:
Hi Apostolos, do you have any recommendation for indentation to continue lines of parameters? For instance, always two tabs?